packages/storybook/src/stories/Typography.stories.tsx

```diff
@@ -10,129 +19,181 @@ const Template: ComponentStory<typeof Typography> = (args) => <Typography {...ar
 
 const TemplateAll: ComponentStory<typeof Typography> = (args) => (
   <>
-    <Typography as="p" {...args}>
-      Paragraph
-    </Typography>
-
-    <Typography variant="caption1" {...args}>
-      Caption 1
-    </Typography>
-
-    <Typography as="h1" variant="display1" {...args}>
+    <Typography as={HEADING_TYPE.H1} variant={DISPLAY_TYPE.DISPLAY1} {...args}>
```

Comment:
Nit: I don't see any benefit of the variable HEADING_TYPE.
Since we are using typescript, you can just specify possible strings as a type and you will see both autosuggestions and errors in case you entered the wrong type.

But since you've added it, we can keep it.